base/applications/winver/winver_p.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:    ReactOS Version Program
+ * LICENSE:    LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:    Main header
+ * COPYRIGHT:  Copyright 2025 Thamatip Chitpong <thamatip.chitpong@reactos.org>
+ */
+
+#pragma once
+
+#include <stdarg.h>
+#include <stdlib.h>
+
+#include <windef.h>
+#include <winbase.h>
+#include <winreg.h>
+#include <winuser.h>
+#include <commctrl.h>
+#include <shellapi.h>
+
+#include <strsafe.h>
+
+#include "resource.h"
+
+#define CONST_STR_LEN(str) (_countof(str) - 1)
+
+typedef struct _WINVER_OS_INFO
+{
+    LPWSTR pszName;
```

Comment:
Should we use dynamic length strings or not?

Reply:
In other words, are 300 characters enough for version info text?

Reply:
> Should we use dynamic length strings or not?

Maybe not, I'll fix this later.

Reply:
Done.